### ISSUE TITLE
About Redfish Interoperability Profile Specification v1.6.0

### ISSUE THREAD

**Issue by vincent-tang1** [DMTF/Redfish-Interop-Validator#176]
Following this spec v1.6.0
not sure if current this repo already supported "Use Cases" situation ?

**Comment by mraineri** [comment]
No, it's not in the tool yet. Pull request #173 will be adding this, but there are issues that need to be resolved first.

**Comment by mraineri** [comment]
PR with UseCase support merged
